.nais/deploy-sandbox.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: pensjonskalkulator-frontend-sandbox
+  namespace: pensjonskalkulator
+  labels:
+    team: pensjonskalkulator
+  annotations:
+    nais.io/run-as-user: "101" # nginx
+spec:
+  image: "{{ image }}"
+  ingresses:
+    - https://www.ekstern.dev.nav.no/pensjon/kalkulator-sandbox
```

Comment:
Dette betyr vel at man må legge til `-sandbox` i [nginx.conf](https://github.com/navikt/pensjonskalkulator-frontend/blob/main/.nginx/nginx.conf#L31), dvs. ha en egen `nginx.conf`-fil for sandbox.
Burde man isteden brukt f.eks. `https://sandbox.ekstern.dev.nav.no/pensjon/kalkulator` (hvis mulig)?

Reply:
Jeg prøver meg på det